### DISCUSSION TITLE
Logstash and ElastAlert on Security Onion 2

### DISCUSSION THREAD

**Discussion by ericchiles** [Security-Onion-Solutions/securityonion#6096] [Unsupported Versions]
Hello,

I got somewhat familiar with using logstash .conf files to manipulate data and add tags on Security Onion 1. I used some of the data manipulation to create elastalert rules that would send slack and email messages based off of things like bad login attempts and non-whitelisted MAC addresses. My goal is to mimic this functionality in SO2, but it seems quite a bit more complex to make these changes. Ideally, I would just drop my elastalert .yaml files and logstash .conf files to a new location and away we go, but that doesn't seem to be the case. Is there some new tool or functionality that I need to learn in order to make custom slack/email alerts and to alter how logs get parsed through logstash? I'm sure I will be able to figure it out, I'm just having difficulties figuring out where to start.

Thanks in advance,
Eric

**Comment by dougburks** [comment]
Have you checked the documentation?
https://docs.securityonion.net/en/2.3/logstash.html#parsing
https://docs.securityonion.net/en/2.3/elasticsearch.html#parsing
https://docs.securityonion.net/en/2.3/elastalert.html
https://docs.securityonion.net/en/2.3/playbook.html
